internal/actions/reparent.go

```diff
@@ -34,6 +34,24 @@ func Reparent(
 	tx meta.WriteTx,
 	opts ReparentOpts,
 ) (*ReparentResult, error) {
+	branchMeta, exist := tx.Branch(opts.Branch)
+	if !exist {
+		_, _ = fmt.Fprint(os.Stderr,
+			"  - Adopting a branch ", colors.UserInput(opts.Branch), " to Av", colors.UserInput(opts.NewParent),
+			"\n",
+		)
+		branchMeta.Parent.Name = opts.NewParent
+		branchMeta.Parent.Trunk = opts.NewParentTrunk
+		if !branchMeta.Parent.Trunk {
+			head, err := repo.RevParse(&git.RevParse{Rev: opts.NewParent})
+			if err != nil {
+				return nil, errors.WrapIff(err, "failed to get HEAD of %q", opts.NewParent)
+			}
+			branchMeta.Parent.Head = head
+		}
+		tx.SetBranch(branchMeta)
```

Comment:
I don't think we need to do this `SetBranch` here.

I think it's effectively a no-op right now because we do the `SetBranch` in `reparentWriteMetadata` which is called on every code path (unless there's a conflict... is that an issue? 🙃).

At any rate, I think this is fine (as in it ~~works~~) but it's a little hard to follow.